wallet/src/account/output_cache/mod.rs

```diff
@@ -221,21 +222,92 @@ impl TokenCurrentSupplyState {
     }
 }
 
+pub enum TokenFreezableState {
```

Comment:
If possible, please always use False=0, and True=1. In many instances I see the opposite, which is confusing for no good reason. For example the IsTokenUnfreezable enum.

Reply:
changed the order